apps/explorer/src/api/tenderly/tenderlyApi.ts

```diff
@@ -14,59 +13,58 @@ import {
 } from './types'
 import { abbreviateString } from 'utils'
 import { SPECIAL_ADDRESSES } from '../../explorer/const'
+import { mapSupportedNetworks, SupportedChainId } from '@cowprotocol/cow-sdk'
 
 export const ALIAS_TRADER_NAME = 'Trader'
 const COW_PROTOCOL_CONTRACT_NAME = 'GPv2Settlement'
-const API_BASE_URLs = _urlAvailableNetwork()
 
-function _urlAvailableNetwork(): Partial<Record<Network, string>> {
-  const urlNetwork = (_networkId: Network): string => `${TENDERLY_API_URL}/${_networkId}`
+const API_BASE_URLs: Record<SupportedChainId, string> = mapSupportedNetworks(
+  (_networkId: SupportedChainId): string => `${TENDERLY_API_URL}/${_networkId}`
```

Comment:
Nitpick: why the `_` prefix?

Reply:
<img width="966" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/ea24df16-e20a-4421-8d2a-4fb8b0f19459">

I have no idea :) Some other developer added it

Reply:
Oh ok. Mind updating it?

Reply:
<img width="1037" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/3348fd61-7e9a-4eae-aaeb-b61491481104">

There are many variables with the prefix, let's do the refactor in another PR?

Reply:
Ok